src/mesh/simple_meshes.jl

```diff
@@ -1,23 +1,212 @@
+function findline(name, lines)
+    for (i, line) in enumerate(lines)
+        if line == name
+            return i
+        end
+    end
+end
+
+struct MeshOptions
+    grouping::Bool
+    remapGroupName::Bool
+end
+
+"""
+ returns the number of elements in a .msh file of a specified dimension
+"""
+function getNumElements(lines,Dim=2)
+    elem_start = findline("\$Elements",lines) + 1
+    temp_numBlocks,_,_,_ = split(lines[elem_start])
+    numBlocks = parse(Int,temp_numBlocks)
+    numElements = 0
+    block_data_line = elem_start + 1
+    for i in 1:numBlocks
+        temp_blockDim,_,_,temp_num_elem_in_block = split(lines[block_data_line])
+        blockDim = parse(Int,temp_blockDim)
+        num_elem_in_block = parse(Int,temp_num_elem_in_block)
+        if blockDim == Dim
+            numElements = numElements + num_elem_in_block
+        end
+        block_data_line = block_data_line + num_elem_in_block + 1
+    end
+    return numElements
+end
+"""
+GMSH uses integers for naming conventions. This function remaps the gmsh ids to
+a list of ids 1:numGroups. This just cleans up a little after Gmsh
+"""
+function remapElementGrouping!(eg)
```

Comment:
```suggestion
function remap_element_grouping!(eg)
```
see above note on the Julia style guide